doc/content/bib/tmap8.bib

```diff
@@ -141,8 +141,16 @@ @article{fowler1977tritium
   publisher={Wiley Online Library}
 }
 
+@book{Carslaw1959conductionadiabatic,
+  title={Conduction of Heat in Solids},
+  author={Carslaw, H. S.  and Jaeger, J. C.},
+  pages={54},
+  year={1959},
+  publisher={2nd Edition, Oxford University Press}
+}
+
```

Comment:
```suggestion
```
I don't think we should have different entries for different pages of the same book.